package.json

```diff
@@ -38,6 +37,7 @@
     "eslint-plugin-jest": "^24.1.5",
     "grunt": "*",
     "grunt-chokidar": "1.0.2",
+    "babelify": "10.0.0",
```

Comment:
cc @tgolen since you have more context than me. It seems like we removed this `devDependency` since it was [deprecated ](https://github.com/Expensify/expensify-common/pull/290/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.sh#r521544886), but its removal is preventing us from properly running `npm link` for Web-E and Web-S, which significantly hinders development when testing out expensify-common changes.

I just wanted to make sure that this wouldn't introduce any breaking changes (and even if it did, it would just be on dev).

Reply:
It _probably_ won't break anything to put this back, but I'm curious what exactly the problem is with `npm link`. Is it throwing an error of some kind?

Reply:
Ah, thanks! Sorry, I missed it above. I guess there isn't much we can do about it. I'm not sure why it's using babelify from that repo, but since this is just a dev dependency, I don't think it's a big deal.